app/trip/components/trip-card.tsx

```diff
@@ -42,15 +40,14 @@ export const TripCard = ({ data }: TripCardProps) => {
       borderBottom={{ base: '1px', md: 'none' }}
       borderColor={{ base: borderColor, md: 'none' }}
       bgColor={{ base: bgColorSp, md: bgColorPc }}
-      align="center"
       _hover={{
         textDecoration: 'none'
       }}
     >
       <Box overflow="hidden" w={{ base: '', md: '100%' }}>
         <Image
-          src={data.image_storage_object_id}
-          alt={`Picture of ${data.title}`}
+          src={data.node.image_storage_object_id || imageSrc}
```

Comment:
Thank you! but in this case I think `||` is better because it could be possible the storage object has empty string?  I wanted to handle any falsy value.

Reply:
> it could be possible the storage object has empty string

No, it's not gonna be empty string, cuz this value is supposed to be UUID. But either way, this will work correctly so it's ok ~!